functions/composer-storage-trigger/index.js

```
@@ -50,23 +50,28 @@ exports.triggerDag = function triggerDag(event) {
   const BODY = {conf: JSON.stringify(event.data)};
 
   // Make the request
-  return authorizeIap(CLIENT_ID, PROJECT_ID, USER_AGENT).then(iap => {
+  try {
+    const iap = await authorizeIap(CLIENT_ID, PROJECT_ID, USER_AGENT);
+
     return makeIapPostRequest(
       WEBSERVER_URL,
       BODY,
       iap.idToken,
       USER_AGENT,
       iap.jwt
     );
-  });
+  } catch (err) {
```

It's cleaner if we throw instead of returning a rejected promise:

```
} catch (err) {
  throw new Error(err);
}
```